stack_master/pbl_config/src/pbl_config/__init__.py

```
@@ -0,0 +1,20 @@
+from pbl_config.CarConfig import CarConfig, load_car_config_ros
```

Ruff (0.8.2)

`pbl_config.CarConfig.CarConfig` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)